lib/entities/quote/DutchQuote.ts

```diff
@@ -426,8 +426,12 @@ export class DutchQuote implements IQuote {
     return this.portionAmountOutStart.mul(this.amountInStart).div(this.amountOutStart.add(this.portionAmountOutStart));
   }
 
+  public get amountInGasAdjusted(): BigNumber {
```

Comment:
isnt this a different answer depending on if its exactIn or exactOut?

Reply:
Well, `amountInGasAdjusted` is only for exact out, although this nomenclature is weird in URA https://github.com/Uniswap/unified-routing-api/blob/main/lib/entities/quote/DutchQuote.ts#L341